### PR TITLE
Add stream selection info action button

### LINKED ISSUE #3045: Meta selection summary only available via delete/reprocess actions
The selection summary dialog provides useful information such as the number of selected items, stream type counts and creation date range. Currently this dialog is displayed when either the delete or reprocess actions are invoked by the user.

This is less than ideal, as in the case of the delete action, the user has already confirmed (twice) that they want to delete all selected items.

A separate action should be added to display summary info.

### PR DESCRIPTION
**Pull request by p-kimberley**
Adds an "info" button to the meta presenter view, to display a selection summary similar to what is shown on `delete` or `reprocess`.

Additionally, the number format has been tweaked for readability, to use the thousands separator (`,`).

Closes #3045 

Screenshot:
image

### CODE REVIEW

**Comment by p-kimberley** [comment]
Closes #3045

File: stroom-core-client/src/main/java/stroom/data/client/presenter/MetaPresenter.java [hunk 1]
```diff
@@ -128,6 +129,9 @@ public MetaPresenter(final EventBus eventBus,
             streamRelationListRestore.setTitle("Restore");
         }
 
+        // Selection information
+        streamListInfo = metaListPresenter.add(SvgPresets.INFO);
```
Comment (at055612) [inline stroom-core-client/src/main/java/stroom/data/client/presenter/MetaPresenter.java:133]: I would set the button title to something like `Selection Summary` to distinguish it from the Info sub-tab of the bottom pane.

File: stroom-core-client/src/main/java/stroom/data/client/presenter/SelectionSummaryPresenter.java [hunk 2]
```diff
@@ -76,8 +77,12 @@ public void onHide(final boolean autoClose, final boolean ok) {
     private void update(final String postAction, final String action, final SelectionSummary result) {
         final SafeHtmlBuilder sb = new SafeHtmlBuilder();
```
Comment (at055612) [inline stroom-core-client/src/main/java/stroom/data/client/presenter/SelectionSummaryPresenter.java:78]: For info you can use TooltipUtil for this sort of HTML building.  Most of our multi-line tooltips are built with that. Feel free to add to it if there is anything extra you need it to do.

Comment (p-kimberley) [reply stroom-core-client/src/main/java/stroom/data/client/presenter/SelectionSummaryPresenter.java:78]: @at055612 Thanks - will keep that in mind.

**Pull request merged**